src/testIndex.ts

```diff
@@ -0,0 +1,11 @@
+const favoriteFruits: string[] = ['apples', 'strawberry', 'orange'];
```

Comment:
TS has powerful inference system: most of the times you don't need to specify type explicitly. Try to remove `: string[]`, hover on that variable and check the type - it'll be the same. 
TS looks for a context, and here is an array of strings.